File: python/7 kyu/322.py
# https://www.codewars.com/kata/5258b272e6925db09900386a/train/python

# Passed

def numbers(*args):
    result = all(type(arg) in [int, float] for arg in args)
    return result

output = numbers(1, 2, 3, 4)
print(output)

File: nodejs/7 kyu/322.js
// https://www.codewars.com/kata/5a19226646d843de9000007d/train/javascript

// Passed

function countConsonants(str) {
  let vowels = new Set("aeiou");
  let uniqueLowercasedCharacters = Array.from(
    new Set(str.toLowerCase().split("")),
  );
  let consonants = uniqueLowercasedCharacters.filter(
    (char) =>
      97 <= char.charCodeAt() && char.charCodeAt() <= 122 && !vowels.has(char),
  );
  let result = consonants.length;
  return result;
}

const output = countConsonants("Count my unique consonants!!");
console.log(output);